docs/design/design-patterns/cloud-resource-design-reference/README.md

```diff
@@ -0,0 +1,17 @@
+# Cloud Resource Design Reference
+
+As cloud usage scales, considerations for subscription design and resource naming/tagging conventions have an impact on governance, operations management, and adoption patterns.
```

Comment:
I would consider creating a new folder for cloud resource design and put this file inside. 
I think there can be more design patterns for cloud env.